src/NHibernate/Loader/Loader.cs

```diff
@@ -379,12 +379,12 @@ internal static EntityKey GetOptionalObjectKey(QueryParameters queryParameters,
 			}
 		}
 
-		internal object GetRowFromResultSet(DbDataReader resultSet, ISessionImplementor session,
-											QueryParameters queryParameters, LockMode[] lockModeArray,
-											EntityKey optionalObjectKey, IList hydratedObjects, EntityKey[] keys,
-											bool returnProxies, IResultTransformer forcedResultTransformer,
-											QueryCacheResultBuilder queryCacheResultBuilder,
-		                                    Action<IEntityPersister, CachePutData> cacheBatchingHandler)
+		public object GetRowFromResultSet(DbDataReader resultSet, ISessionImplementor session,
+		                                  QueryParameters queryParameters, LockMode[] lockModeArray,
+		                                  EntityKey optionalObjectKey, IList hydratedObjects, EntityKey[] keys,
+		                                  bool returnProxies, IResultTransformer forcedResultTransformer,
+		                                  QueryCacheResultBuilder queryCacheResultBuilder,
+		                                  Action<IEntityPersister, CachePutData> cacheBatchingHandler)
```

Comment:
That formatting may not be kept depending on the developer IDE (VS's formatting vs Rider, ...). For avoiding new undue noise at next change, it is better to format this with tab only, like so:

```suggestion
		public object GetRowFromResultSet(
			DbDataReader resultSet, ISessionImplementor session,
			QueryParameters queryParameters, LockMode[] lockModeArray,
			EntityKey optionalObjectKey, IList hydratedObjects, EntityKey[] keys,
			bool returnProxies, IResultTransformer forcedResultTransformer,
			QueryCacheResultBuilder queryCacheResultBuilder,
			Action<IEntityPersister, CachePutData> cacheBatchingHandler)
```

Reply:
OK. Do you prefer one parameter each line like 
```
		public void InitializeEntitiesAndCollections(
			IList hydratedObjects, 
			DbDataReader reader, 
			ISessionImplementor session, 
			bool readOnly,
			CacheBatcher cacheBatcher)
```
, or multiple ones?
```
public object GetRowFromResultSet(
			DbDataReader resultSet, ISessionImplementor session,
			QueryParameters queryParameters, LockMode[] lockModeArray,
			EntityKey optionalObjectKey, IList hydratedObjects, EntityKey[] keys,
			bool returnProxies, IResultTransformer forcedResultTransformer,
			QueryCacheResultBuilder queryCacheResultBuilder,
			Action<IEntityPersister, CachePutData> cacheBatchingHandler)
```

Reply:
The later, at least when there are that many parameters.